src/components/List/List.jsx

```diff
@@ -15,7 +15,15 @@ const List = () => {
     <div>
       {error && <strong>Error: {JSON.stringify(error)}</strong>}
       {loading && <span>Collection: Loading...</span>}
-      {value && (
+
+      {value && value.empty && (
+        <p>
+          There are not items in this list yet. You can add items with the "Add
```

Comment:
Super tiny grammatical fix: "There are not _any_ items..." or "There are _no_ items..."